hooks/500-create-xdg.chroot

```diff
@@ -29,6 +29,8 @@ x-scheme-handler/mailto=xdg-open.desktop
 x-scheme-handler/help=xdg-open.desktop
 EOF
 
+cp /usr/bin/xdg-open /usr/bin/xdg-email
```

Comment:
could it have been a symlink?

Reply:
Also, it does not look like they are the same binary in fact, not in noble.

Reply:
Oh, but I see `xdg-open` is a script we create in this hook nevermind.